pymc3/distributions/multivariate.py

```diff
@@ -1646,16 +1650,21 @@ def dist(
         rowchol=None,
         colcov=None,
         colchol=None,
-        shape=None,
         *args,
         **kwargs,
     ):
 
         cholesky = Cholesky(lower=True, on_error="raise")
 
-        if mu.ndim == 1:
-            raise ValueError(
-                "1x1 Matrix was provided. Please use Normal distribution for such cases."
+        if "size" in kwargs and kwargs["size"] is not None:
+            raise NotImplementedError("MatrixNormal doesn't support size argument")
+
+        if "shape" in kwargs:
```

Comment:
Should add a test for shape and I think it is allowed to be None as well (like `size`).